src/main/java/survey/NumberQuestion.java

```diff
@@ -0,0 +1,26 @@
+package survey;
+
+import javax.persistence.Entity;
+import java.util.ArrayList;
+
+@Entity
+public class NumberQuestion extends Question {
+
+    private ArrayList<String> answers = new ArrayList<>();
+
+    /**
+     * Default constructor
+     */
+    public NumberQuestion() {
+        super();
+    }
+
+    /**
+     * Constructor with specified question
+     * @param question
+     */
+    public NumberQuestion(String question, String min, String max) {
```

Comment:
Since min and max are specific to the Number question, they should only be in the NumberQuestion class, not in the parent abstract class.
Is there are reason you put them in the parent class? Did it cause the repository to break?

Reply:
To clarify, it is more okay for the message to the server to have min and max be in the same message class, cause it makes everything simpler for parsing. But it should be avoided if possible for the database.

Reply:
No reason, I do agree the boundaries are specific to the numberical type. Will move it